Cybereason Fetch Incidents Fix (#21677)
demisto/content: Packs/Cybereason/ReleaseNotes/1_0_23.md

#### Integrations
##### Cybereason

> - Fixed an issue with fetch incidents last run time.

Reviewer: ```suggestion
- Fixed an issue where **fetch-incidents** failed to run.
```
